arrow-core-data/src/main/kotlin/arrow/typeclasses/TraverseFilter.kt

```diff
@@ -30,16 +28,23 @@ interface TraverseFilter<F> : Traverse<F>, FunctorFilter<F> {
    * Returns [F]<[B]> in [G] context by applying [AP] on a selector function [f], which returns [Option] of [B]
    * in [G] context.
    */
+  @Deprecated("Please use traverseFilterNullable")
   fun <G, A, B> Kind<F, A>.traverseFilter(AP: Applicative<G>, f: (A) -> Kind<G, Option<B>>): Kind<G, Kind<F, B>>
 
+  fun <G, A, B> Kind<F, A>.traverseFilterNullable(AP: Applicative<G>, f: (A) -> Kind<G, B?>): Kind<G, Kind<F, B>>
```

Comment:
does `B?` make sense here :thinking:

Reply:
Yes, it does. This works like `mapNotNull` but for `Traverse` instead of `FunctorFilter`, so perhaps `traverseNotNull` would be a better name?